docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx

```diff
@@ -193,6 +202,23 @@ This should result in a config file like the one below:
 ```toml
 (!examples/resources/plugins/teleport-slack.toml!)
 ```
+</TabItem>
+<TabItem label="Helm Chart">
+The Slack Helm Chart uses a YAML values file to configure the plugin.
+On the host where you've helm installed, create a file called
```

Comment:
```suggestion
On your local workstation, create a file called
```
Unless users would likely be running Helm on a remote host. In that case, we should change "where you've helm installed" to "where you've installed Helm".